tests/Hedgehog.Benchmarks/GenBenchmarks.fs

```diff
@@ -0,0 +1,26 @@
+namespace Hedgehog.Benchmarks
+
+open BenchmarkDotNet.Attributes
+open BenchmarkDotNet.Jobs
+
+module HRange = Hedgehog.Range
+module HGen = Hedgehog.Gen
+module FGen = FsCheck.Gen
+
+[<SimpleJob(RuntimeMoniker.NetCoreApp31)>]
+type GenBenchmarks () =
+
+    [<Benchmark>]
+    member __.GenSampleLargeHedgehog () =
+        HRange.constant -100 100
+        |> HGen.int32
+        |> HGen.sample 100 100000
+        |> Seq.iter ignore
+
+    [<Benchmark>]
+    member __.GenSampleLargeFsCheck () =
+        ['0'..'9'] @ ['A'..'Z'] @ ['a'..'z']
+        |> FGen.elements
+        |> FGen.listOfLength 10
+        |> FGen.map (List.toArray >> System.String.Concat)
+        |> FGen.sample 0 10000
```

Comment:
This looks like you are comparing apples to oranges.  Most surprisingly, it looks to me like you are sampling from the FsCheck generator 10,000 times but sampling from the Hedgehog generator 100,000 times.

Can you add to the method names what is being sample and how many times?

Reply:
🤦🏼‍♂️ Good catch! I got my wires crossed with the two examples in #135, this has been fixed. If anything, it gives us even more room for improvement 😅 

``` ini
BenchmarkDotNet=v0.13.1, OS=debian 10 (container)
Intel Core i7-4790K CPU 4.00GHz (Haswell), 1 CPU, 8 logical and 4 physical cores
.NET Core SDK=3.1.409
  [Host]        : .NET Core 3.1.15 (CoreCLR 4.700.21.21202, CoreFX 4.700.21.21402), X64 RyuJIT DEBUG
  .NET Core 3.1 : .NET Core 3.1.15 (CoreCLR 4.700.21.21202, CoreFX 4.700.21.21402), X64 RyuJIT

Job=.NET Core 3.1  Runtime=.NET Core 3.1  
```
|                 Method |      Mean |    Error |   StdDev |
|----------------------- |----------:|---------:|---------:|
| GenSampleLargeHedgehog | 125.95 ms | 0.336 ms | 0.298 ms |
|  GenSampleLargeFsCheck |  26.18 ms | 0.237 ms | 0.210 ms |